projects/kit/components/slider/slider.component.ts

```diff
@@ -75,20 +81,26 @@ export class TuiSliderComponent {
         return isEdgeOlderThan(CHROMIUM_EDGE_START_VERSION, this.userAgent);
     }
 
-    get value(): number {
-        const {control} = this;
-        const controlValue =
-            control instanceof NgModel ? control.viewModel : control?.value;
-
-        return controlValue || Number(this.elementRef.nativeElement.value) || 0;
-    }
-
     constructor(
         @Optional()
         @Self()
         @Inject(NgControl)
-        private readonly control: NgControl | null,
+        readonly control: NgControl | null,
```

Comment:
Why `control` is no longer `private`? I think you meant to just remove all modifiers. And `ChangeDetectorRef` is also not needed on the class. I prefer `watch(changeDetectorRef)` operator in the chain and empty `subscribe` but that's up to you.

Reply:
fixed